content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- Rust-first: with all the safety, tooling and power that comes with a modern systems language
```

Comment:
I think bullet points should have consistent formatting. In this case, lets start each one with a captial letter.